tools/importer/rules/metadata.js

```diff
@@ -41,7 +40,7 @@ export default function createMetadataBlock(main, document) {
       }
     }
   });
-
+  meta.template = 'helpxmain-article'; // make sure that's there
```

Comment:
Minor: You may want to add a reason why field is required (2 templates, column-wise difference, etc.).